geode/src/main/java/org/apache/calcite/adapter/geode/rel/GeodeSchemaFactory.java

```diff
@@ -39,18 +45,58 @@
   public static final String PDX_SERIALIZABLE_PACKAGE_PATH = "pdxSerializablePackagePath";
   public static final String ALLOW_SPATIAL_FUNCTIONS = "spatialFunction";
   public static final String COMMA_DELIMITER = ",";
+  public static final String GEODE_JNDI_INITIAL_CONTEXT_FACTORY =
+      "org.apache.geode.internal.jndi.InitialContextFactoryImpl";
+  public static final String GEODE_CLIENT_CACHE_OBJECT_KEY = "geodeClientCacheObjectKey";
 
   public GeodeSchemaFactory() {
     // Do Nothing
   }
 
+  private static Context getContext() {
+    Context context = null;
+
+    try {
+      Hashtable env = new Hashtable();
+      env.put(Context.INITIAL_CONTEXT_FACTORY, GEODE_JNDI_INITIAL_CONTEXT_FACTORY);
+      context = new InitialContext(env);
+    } catch (Exception ignored) {
```

Comment:
can we just catch [NamingException](https://docs.oracle.com/javase/7/docs/api/javax/naming/NamingException.html) here ?